src/etc/inc/interfaces.inc

```diff
@@ -677,30 +677,23 @@ function interfaces_bridge_configure($checkmember = 0, $realif = "") {
 			if (empty($bridge['bridgeif'])) {
 				$bridge['bridgeif'] = "bridge{$i}";
 			}
-			if (!empty($realif) && $realif != $bridge['bridgeif']) {
+			if (!empty($realif) && ($realif != $bridge['bridgeif'])) {
 				continue;
 			}
-
-			if ($checkmember == 1) {
-				/* XXX: It should not be possible no? */
-				if (strstr($bridge['if'], '_vip')) {
-					continue;
-				}
-				$members = explode(',', $bridge['members']);
-				foreach ($members as $member) {
-					if (!empty($config['interfaces'][$bridge['if']]) && $config['interfaces'][$bridge['if']]['ipaddrv6'] == "track6") {
-						continue 2;
-					}
+			$ifname = false;
+			foreach ($config['interfaces'] as $intname => $intpar) {
+				if ($intpar['if'] == $bridge['bridgeif']) {
+					$ifname = $intname;
```

Comment:
Are you looking for last item or array?  If not, it would be better to add a `break;` here

Reply:
thanks, fixed

for some reason PR is closed after I did `git push -f origin`,
new PR https://github.com/pfsense/pfsense/pull/4360